GraalVM/CE/Dockerfile.ol8

```diff
@@ -0,0 +1,37 @@
+# LICENSE UPL 1.0
+#
+# Copyright (c) 2015 Oracle and/or its affiliates. All rights reserved.
```

Comment:
The date should be 2020 and please remove "All rights reserved." too. You also need to add a second line, as follows: 
```
Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/
```